cloudant-client/src/main/java/com/cloudant/client/internal/query/Helpers.java

```diff
@@ -59,8 +59,8 @@ public static String quoteNoSquare(Object[] os) {
         return quoteInternal(os, ", ", "", "", "", "");
     }
 
-    public static String quoteCurly(Object[] os) {
-        if (os.length == 1) {
+    public static String quoteCurly(Object[] os, String op) {
```

Comment:
As this is a public method, do you think you could add re-add the original `public static String quoteCurly(Object[] os)` so that we don't have to do a major release for this change? (I understand that it says `internal` in the path, but I wonder if there's a way to retain backwards compatibility.)

Reply:
I reverted the original method: https://github.com/cloudant/java-cloudant/pull/516/files#diff-b2d6eda22288fdd2ad1e1819d87181ecR70-R76

Reply:
IMO this was unnecessary it is not API and so would not have required a major release, but I can understand treading carefully and I'm not strongly opposed to the duplication.